extension/vsc-js/package.json

```diff
@@ -0,0 +1,40 @@
+{
+	"name": "howdoi",
```

Comment:
There's some tabs in this file. Also should we call this `howdoi-vscode` or something like that?